### PR TITLE
Replace variables names in runtime

### PR DESCRIPTION
**Pull request by alxpy**
I started implement `transform our translations object (json with translations) in form of Hello ${ username } to Hello ${ 0 } in the runtime`
ref -- https://github.com/c-3po-org/c-3po/issues/20

### CODE REVIEW

**Comment by alxpy** [comment]
@AlexMost @MrOrz so, I think I'm done (migration script is in progress...), please review my code (this PR and https://github.com/c-3po-org/babel-plugin-c-3po/pull/71).

**Comment by alxpy** [comment]
@MrOrz can you try/test this patch too?

File: src/loader.js [hunk 1]
```diff
@@ -3,6 +3,53 @@ import path from 'path';
 import gParser from 'gettext-parser';
```
Comment (AlexMost) [inline src/loader.js]: file `loader.js` is used on a backend. We should be able to transform our translations object on a frontend. I think we should put all transform logic inside `addLocale` method.

File: src/loader.js [hunk 2]
```diff
@@ -3,6 +3,53 @@ import path from 'path';
...
+        newTranslate.msgstr.push(replaceVariables(str, variableNumberMap));
+    }
+    newTranslate.comments = translate.comments;
+    return newTranslate;
+}
+
+function transformTranslateObj(translateObj) {
+    const newTranslations = {};
+    const translations = translateObj.translations[''];
+    for (const msgid of Object.keys(translations)) {
```
Comment (AlexMost) [inline src/loader.js]: Consider that `Object.keys` doesn't work in IE<9 and in some other old browsers.

File: src/loader.js [hunk 3]
```diff
@@ -3,6 +3,53 @@ import path from 'path';
...
+        const newTranslate = transformTranslate(translations[msgid]);
+        newTranslations[newTranslate.msgid] = newTranslate;
+    }
+    translateObj.translations[''] = newTranslations;
```
Comment (AlexMost) [inline src/loader.js]: There is a plan to support contexts. So `translations['']` with empty string is just a default context. We should iterate through `translateObj.translations` keys and reassign them with new translations, but not hard code them.

File: src/loader.js [hunk 4]
```diff
@@ -16,13 +63,22 @@ export function loadFile(filepath) {
     throw new Error(`Unsupported filetype ${filepath}`);
 }
 
-export function loadLocale(locale, filepath) {
+export function loadLocale(locale, filepath, replaceVariablesNames = false) {
     if (typeof filepath === 'string') {
```
Comment (AlexMost) [inline src/loader.js]: I think it makes sense to move replaceVariablesNames logic to `addLocale` method.

Comment (alxpy) [inline src/loader.js]: done

**Comment by AlexMost** [comment]
LGTM

**Pull request merged**